plugin/src/main/java/com/denizenscript/denizen/objects/EntityTag.java

```diff
@@ -4486,6 +4483,24 @@ else if (getBukkitEntity() instanceof Creeper) {
             ((Lootable) getBukkitEntity()).setLootTable(table);
         }
 
+        // <--[mechanism]
+        // @object EntityTag
+        // @name is_sleeping
+        // @input EntityTag
+        // @description
+        // Sets whether a fox is sleeping.

```

Comment:
there should be more text here, like that has_ai thing mentioned in the discord thread